lib/_http_outgoing.js

```diff
@@ -122,12 +121,9 @@ function OutgoingMessage() {
   this._contentLength = null;
   this._hasBody = true;
   this._trailer = '';
-  this[kNeedDrain] = false;
 
-  this.finished = false;
   this._headerSent = false;
   this[kCorked] = 0;
```

Comment:
Is this still necessary after the swtich to `_writableState.corked`?

Reply:
Unfortunately yes. I have a future todo for this.